.github/workflows/test-template.yml

```diff
@@ -102,7 +102,7 @@ jobs:
   # CLIENT (functional and legacy tests without types test)
   #
   client:
-    name: Client (functional and legacy tests without types test)
+    name: Client (func+legacy-types)
```

Comment:
I read this as "legacy-types" :/

Reply:
It needs to be somehow shorter to be readable from the UI unfortunately...

Reply:
```suggestion
    name: Client func/legacy notypes
```
maybe that makes it

Reply:
Yeah, I get that. But still clear and concise.